hoomd/hpmc/integrate.py

```diff
@@ -645,53 +687,54 @@ class Polyhedron(_HPMCIntegrator):
-        deterministic (bool): Make HPMC integration deterministic on the GPU
-
     Polyhedron parameters:
 
-    * *vertices* (**required**) - vertices of the polyhedron as is a list of
-      (x,y,z) tuples of numbers (distance units)
-
-        * The origin **MUST** strictly be contained in the generally nonconvex
-          volume defined by the vertices and faces
-        * The (0,0,0) centered sphere that encloses all vertices should be of
-          minimal size for optimal performance (e.g. don't translate the shape
-          such that (0,0,0) right next to a face).
-
-    * *faces* (**required**) - a list of vertex indices for every face
-
-        * For visualization purposes, the faces **MUST** be defined with a
-          counterclockwise winding order to produce an outward normal.
-
-    * *sweep_radius* (**default: 0.0**) - rounding radius applied to polyhedron
-    * *ignore_statistics* (**default: False**) - set to True to disable ignore
-      for statistics tracking
-    * *overlap* (**default: None for all particles**) - only check overlap
-      between constituent particles for which *overlap [i] & overlap[j]* is !=0,
-      where '&' is the bitwise AND operator.
-
-    * *capacity* (**default: 4**) - set to the maximum number of particles per
-      leaf node for better performance
-
-        * .. versionadded:: 2.2
-
-    * *origin* (**default: (0,0,0)**) - a point strictly inside the shape,
-      needed for correctness of overlap checks
-
-        * .. versionadded:: 2.2
-
-    * *hull_only* (**default: True**) - if True, only consider intersections
-      between hull polygons
-
-        * .. versionadded:: 2.2
+    shape (particle type, dict): defines the shape of the object.
+
+        Keys:
+            * *vertices* (list, **required**) - vertices of the polyhedron as a
+              list of (x, y, z) tuples (distance units)
+                * The origin **MUST** strictly be contained in the generally
+                  nonconvex volume defined by the vertices and faces
+                * The (0,0,0) centered sphere that encloses all vertices should
+                  be of minimal size for optimal performance (e.g. don't
+                  translate the shape such that (0,0,0) right next to a face).
+            * *faces* (list, **required**) - list of vertex indices for every
+              face
+                * For visualization purposes, the faces **MUST** be defined with
+                  a counterclockwise winding order to produce an outward normal.
+            * *ignore_statistics* (**default: False**) - set to True to ignore
+              tracked statistics
+            * *sweep_radius* (**default: 0.0**) - radius of the sphere swept
+              around the edges of the polyhedron (distance units). Set a
+              non-zero sweep_radius to create a spheropolyhedron
+            * *overlap* (**default: None**) - only check overlap between
```

Comment:
Does not describe what the default means.

Reply:
fixed.